pypit/ararc.py

```diff
@@ -492,10 +495,10 @@ def simple_calib(slf, det, get_poly=False):
         msarc = slf._msarc[det-1]
         wave = arutils.func_val(fit, np.arange(msarc.shape[0])/float(msarc.shape[0]),
             'legendre', minv=fmin, maxv=fmax)
-        debugger.xplot(xfit,yfit, scatter=True,
-            xtwo=np.arange(msarc.shape[0])/float(msarc.shape[0]),
-            ytwo=wave)
-        debugger.xpcol(xfit*msarc.shape[0], yfit)
+        #debugger.xplot(xfit,yfit, scatter=True,
```

Comment:
Maybe this can just be deleted? (likewise with the commented out code on line 448/449